static/js/settings_org.js

```diff
@@ -877,28 +871,26 @@ exports.build_page = function () {
 
     function populate_data_for_request(subsection) {
         const data = {};
-        const properties_elements = get_subsection_property_elements(subsection);
-
-        for (let input_elem of properties_elements) {
+        get_subsection_property_elements(subsection).forEach(input_elem => {
```

Comment:
If we’re fixing this to be a real array, what’s wrong with `for … of`?

Reply:
I thought to make this loop consistent with `forEach` usage, but I wasn't strong about it either, so yeah this commit is totally optional.